pkg/base1/Makefile.am

```diff
@@ -150,13 +155,17 @@ EXTRA_DIST += \
 	pkg/base1/moment.min.js \
 	pkg/base1/react.min.js \
 	pkg/base1/term.min.js \
+	pkg/base1/cockpit-react.min.js \
 	pkg/base1/test-dbus-common.js \
 	$(base_BUNDLE) \
 	$(base_GEN) \
 	$(base_GEN:.js=.min.js) \
 	$(NULL)
 
-update-lib:: update-bower
+REACT_INPUTS = \
+	pkg/base1/cockpit-react-terminal.js
+
+update-lib:: update-bower $(REACT_INPUTS)
```

Comment:
I'm not sure update-lib is the right target for this. The react components aren't an upstream lib that we want to pull in, but but part of continued development. Needing to call update-lib to hack on this seems very unwieldy. What do you think?